resqpy/olio/triangulation.py

```diff
@@ -658,7 +676,7 @@ def triangulated_polygons(p, v, centres = None):
     t_index = 0
 
     for cell, poly_vertices in enumerate(v):
-        # add polygon centre points to points array, if not already provided
+        # add polygon centre points to point array, if not already provided
```

Comment:
I think 'points' was correct, as in an array of points?

Reply:
yes, 'points' is better